### PR TITLE
Minor UI fixes

### PR DESCRIPTION
**Pull request by Shruti-Apte**
## Notion card
NA
## Summary

- Fixed toast message on new invoice page for not adding client and proceeding
- Address field of add new client
- Timesheet entry scroll issue
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/javascript/src/components/time-tracking/EntryCard.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const EntryCard: React.FC<props> = ({
         <p className="text-lg mx-2">•</p>
         <p className="text-lg">{project}</p>
       </div>
-      <p className="max-h-32 overflow-scroll text-sm text-miru-dark-purple-400 break-words whitespace-pre-wrap">
+      <p className="max-h-32 overflow-auto text-sm text-miru-dark-purple-400 break-words whitespace-pre-wrap">
```
Comment (onkar-saeloun) [inline app/javascript/src/components/time-tracking/EntryCard.tsx:35]: was this overflow issue regarding horizontal scroll, vertical scroll or both?

Comment (Shruti-Apte) [reply app/javascript/src/components/time-tracking/EntryCard.tsx:35]: Both

**Review by onkar-saeloun — APPROVED** [review]
There are some gemfile.lock changes I guess to need to take latest develop pull. 
LGTM apart from that.

**Comment by Shruti-Apte** [comment]
Raised  a different PR for this because unable to revert gemfile.lock changes

**Pull request closed without merging**